src/components/Api/RecipientApi.js

```diff
@@ -1,39 +1,35 @@
+import axios from "axios";
+
 const BASE_URL = "https://rolling-api.vercel.app";
 
 export const getRecipient = async (recipientId) => {
-  const RESPONSE = await fetch(`${BASE_URL}/4-12/recipients/${recipientId}/`);
-
-  if (!RESPONSE.ok) {
-    throw new Error("사용자 정보를 받아오는데 실패했어요.");
-  }
+  const RESPONSE = await axios
+    .get(`${BASE_URL}/4-12/recipients/${recipientId}/`)
```

Comment:
이거 util 파일에 axiosInstance로 불러오면 baseURL 기본 설정 되어있어요!  axios 대신에 그걸로 사용하면 뒤에 recipients... 부분만 넣으면 똑같이 나옵니다~
```suggestion
  const RESPONSE = await axiosInstance
    .get(`recipients/${recipientId}/`)
```
대신 위에 import를 axiosInstance로 바꿔야겠죠?

Reply:
와 설명짱👍🏻👍🏻👍🏻

Reply:
수정했어요 :)